oak-run-commons/src/test/java/org/apache/jackrabbit/oak/index/indexer/document/flatfile/pipelined/PipelinedTreeStoreIT.java

```diff
@@ -364,11 +365,12 @@ public void getOnlyModified() throws Exception {
             contentDamBuilder.child("old.png");
             contentDamBuilder.child("change.png").setProperty("test", 0);
             rwNodeStore.merge(rootBuilder, EmptyHook.INSTANCE, CommitInfo.EMPTY);
-            long oldMod = NodeDocument.getModifiedInSecs(System.currentTimeMillis());
+            Clock clock = rwNodeStore.getClock();
+            long oldMod = NodeDocument.getModifiedInSecs(clock.getTime());
             // wait until the modified time changes
             do {
-                Thread.sleep(10);
-                minModified = NodeDocument.getModifiedInSecs(System.currentTimeMillis());
+                clock.waitUntil(clock.getTime() + 1000);
```

Comment:
Great! I didn't think about using the virtual clock. I wasn't aware waitUntil increments the counter...